appSetting.json

```diff
@@ -1,11 +1,11 @@
 {
     "autoTestConfig":
     {
-        "url":"",
-        "user":"",
-        "password":"",
-        "userFullName" : "",
-        "apiurl": ""
+        "url":"https://fss-dev.admiralty.co.uk",
```

Comment:
Please revert the changes to autoTestConfig

Reply:
Reverted the changes, Thanks Chris.